config/initializers/012-web_hook_events.rb

```diff
@@ -19,7 +19,7 @@
 
 DiscourseEvent.on(:post_edited) do |post, topic_changed|
   WebHook.enqueue_post_hooks(:post_edited, post)
-  WebHook.enqueue_topic_hooks(:topic_edited, post.topic) if post.is_first_post? && topic_changed
+  WebHook.enqueue_topic_hooks(:topic_edited, post.topic) if post.is_first_post? && topic_changed && post.topic
```

Comment:
hmm... same goes for the post_edited hook

it should also check for post.topic, I would not trigger any events if the topic is not there.